### PR TITLE
Update documentation.scala.html

### PR DESCRIPTION
**Pull request by danjee**
Added documentation for Pippo framework

### CODE REVIEW

File: app/views/documentation.scala.html [hunk 1]
```diff
@@ -505,7 +506,33 @@ <h4 style="padding-top: 15px;">Public CDN via <a href="http://www.jsdelivr.com"
...
 
+                The Pippo integration of Webjars is pretty straightforward.
+
+                <h4>Maven Example (<a href="https://github.com/decebals/pippo/tree/master/pippo-demo/pippo-demo-ajax">example app</a>)</h4>
+
+                First you have to let Pippo know you are using webjars. This can be done in <span class="label label-info">Application.onInit method</span>:
+                <pre><code>
+public class AjaxApplication extends Application {
+
+    @Override
```
Comment (jamesward) [inline app/views/documentation.scala.html]: You need to escape the `@` with `@@`.

Comment (danjee) [inline app/views/documentation.scala.html]: I saw it. Thanks

**Comment by jamesward** [comment]
Thanks!

**Comment by danjee** [comment]
You are welcome!

**Pull request merged**